### DISCUSSION TITLE
Must i delete/disable ROOP extension before try this one?

### DISCUSSION THREAD

**Discussion by ZeroCool22** [glucauze/sd-webui-faceswaplab#88] [Q&A] [answered]
Wonder if i could cause some conflict?

**Comment by agarzon** [comment, marked as answer]
You don't NEED, however this extension works as its own, so yo can freely disabled roop (I did)

**Comment by LucianoDaluz** [comment]
If you use Reactor (a Roop without the NSFW lock) it will stop working, if you want it to work, just do it like this:
You can try the following:

    Delete FaceSwapLab from the extensions folder
    Delete onnxruntime and onnxruntime-gpu folders from the site-packages (inside VENV Lib)
    Run SD WebUI and let ReActor to install onnxruntime, close SD WebUI
    git clone https://github.com/glucauze/sd-webui-faceswaplab into the SD WebUI extensions folder
    Edit [sd-webui-faceswaplab/install.py](https://github.com/glucauze/sd-webui-faceswaplab/blob/main/install.py) file:
        Line 9: use_gpu = True -> use_gpu = False
    Run SD WebUI, both extensions must work fine after that

**Reply by glucauze** [reply]
That will disable support for GPU

**Reply by LucianoDaluz** [reply]
Yep, but there is no difference in performance, and it is the only way to make both work. But to be honest, faceswap lab gets way better results than reactor, so there isnt to many reasons to keep both.

**Reply by glucauze** [reply]
In my case, GPU gives way better performance :) but that may vary.

**Reply by KINGLIFER** [reply]
Yeah that did not work for me. Actually the whole thing (Faceswap and roop) are gone. After some tinkering (I just put the folders back) I get this error.
\onnxruntime\core\session\provider_bridge_ort.cc:1131 onnxruntime::ProviderLibrary::Get [ONNXRuntimeError] : 1 : FAIL : LoadLibrary failed with error 126 "" when trying to load "D:\GitHub\stable-diffusion-webui\venv\lib\site-packages\onnxruntime\capi\onnxruntime_providers_tensorrt.dll"

**Reply by LucianoDaluz** [reply]
To be honest Faceswap lab is way better than Roop and Reactor. So no reason to keep the others.

**Comment by KINGLIFER** [comment]
Tried to install a fresh Stab Matrix and it would not install either.  line 14, in <module>
        import insightface
    ModuleNotFoundError: No module named 'insightface'
